tests/ops/service/privacy_request/test_email_batch_send.py

```diff
@@ -0,0 +1,524 @@
+        assert requeue_privacy_requests.called
+
+    @mock.patch(
+        "fides.api.ops.service.connectors.erasure_email_connector.send_single_erasure_email",
+    )
+    @mock.patch(
+        "fides.api.ops.service.privacy_request.email_batch_service.requeue_privacy_requests_after_email_send",
+    )
+    @pytest.mark.usefixtures("privacy_request_awaiting_consent_email_send")
+    @pytest.mark.usefixtures("attentive_email_connection_config")
+    def test_send_email_batch_missing_identities(
+        self,
+        requeue_privacy_requests,
+        send_single_erasure_email,
+    ) -> None:
+        exit_state = send_email_batch.delay().get()
+        assert exit_state == EmailExitState.complete
+
+        assert not send_single_erasure_email.called
+        assert requeue_privacy_requests.called
+
+    @mock.patch(
+        "fides.api.ops.service.privacy_request.email_batch_service.requeue_privacy_requests_after_email_send",
+    )
+    @pytest.mark.usefixtures("attentive_email_connection_config", "test_fides_org")
+    def test_send_erasure_email_failure(
+        self,
+        requeue_privacy_requests,
+        db,
+        privacy_request_awaiting_erasure_email_send,
+    ) -> None:
+        with pytest.raises(MessageDispatchException):
+            # Assert there's no messaging config hooked up so this consent email send should fail
+            MessagingConfig.get_configuration(
+                db=db, service_type=CONFIG.notifications.notification_service_type
+            )
+        identity = Identity(email="customer_1#@example.com")
+        privacy_request_awaiting_erasure_email_send.cache_identity(identity)
+        privacy_request_awaiting_erasure_email_send.save(db)
+
+        exit_state = send_email_batch.delay().get()
+        assert exit_state == EmailExitState.email_send_failed
+
+        assert not requeue_privacy_requests.called
+        email_execution_log: ExecutionLog = ExecutionLog.filter(
+            db=db,
+            conditions=(
+                (
+                    ExecutionLog.privacy_request_id
+                    == privacy_request_awaiting_erasure_email_send.id
+                )
+                & (ExecutionLog.status == ExecutionLogStatus.complete)
+            ),
+        ).first()
+        assert not email_execution_log
+
+    @mock.patch(
+        "fides.api.ops.service.connectors.erasure_email_connector.send_single_erasure_email",
+    )
+    @mock.patch(
+        "fides.api.ops.service.privacy_request.email_batch_service.requeue_privacy_requests_after_email_send",
+    )
+    def test_send_erasure_email(
+        self,
+        requeue_privacy_requests,
+        send_single_erasure_email,
+        db,
+        privacy_request_awaiting_erasure_email_send,
+        second_privacy_request_awaiting_consent_email_send,
+        attentive_email_connection_config,
+    ) -> None:
+        """Verify that a privacy request queued for a consent email doesn't trigger an erasure email."""
```

Comment:
Is this docstring accurate to what you're testing here?

Reply:
Updated the docstring

```
"""
Test for batch erasure email, also verifies that a privacy request
queued for a consent email doesn't trigger an erasure email.
"""
```